calendar/src/Components/EventForm/event.css

```diff
@@ -0,0 +1,93 @@
+.event-wrapper {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  z-index: 999;
+  justify-content: center;
+}
+.event-container {
+  padding: 2.6em;
+  border-radius: 5px;
+  background-color: white;
+  position: relative;
+}
+.event-input {
+  background-color: rgba(255, 255, 255, 0);
+  border: none;
+  font-family: Arial, Helvetica, sans-serif;
+  padding: 0.5em;
+  border-radius: 5px;
+}
+.validationMessage {
+  width: 100%;
+  opacity: 0;
+  transition: opacity 0.3s;
+  padding: 0.5em;
+  border-radius: 5px;
+  margin: 0.3em 0;
+}
+.invalidInput {
+  background-color: var(--error-message-bg);
+  opacity: 1;
+}
+.eventInputContainer {
+  width: 100%;
+}
+.event-title {
+  width: 100%;
+}
+.event-titleInput {
+  border: none;
+  border-bottom: 1px solid var(--grey_light);
+  width: 100%;
+}
+
+.event-controls {
+  justify-content: right;
+}
+.display-none {
+  display: none;
+}
+
+.event-left {
+  width: 1.6em;
+  margin-right: 1em;
+  align-self: start;
+}
+.event-left > img {
+  width: 100%;
+  opacity: 0.5;
+  margin-top: 0.3em;
+}
+.event-right {
+  display: flex;
+  flex-grow: 1;
+  border-radius: 4px;
+  flex-direction: column;
+}
+.eventInputContainer {
+  border-radius: 5px;
+}
+.eventInputContainer:hover {
+  background-color: var(--grey_light);
+}
+.event-textarea {
+  border: 1px solid var(--grey_light);
+  width: 100%;
+}
+.button-cancel {
+  margin-right: 1em;
+}
+.button-save {
+  background-color: var(--primary);
+  color: white;
+}
+.button-save:hover {
+  background-color: var(--primary_dark);
+}
+input[type="time"]::-webkit-calendar-picker-indicator {
```

Comment:
Style will not work on firefox. Webkit selectors are browser specific, but we try to write code that works on all browsers.